### PR TITLE
Adjust models.yml

### LINKED ISSUE #5590: Rename some fields in the model.yml
- `HTML` and `HTMLVideo` should be named `HTMLStrict` and `HTMLPermissive` as @normanjaeckel already suggested, since the point will most certainly come when we add tags to the permissive type and then the name doesn't fit
- `datetime` should be called `timestamp` since we save it as a timestamp

Feel free to add other suggestions here.

### PR DESCRIPTION
**Pull request by jsangmeister**
fixes #5590 

Adds `on_delete` fields to the model description. Still missing on many fields, but enough for first testing purposes. Rest can be added later.

### CODE REVIEW

File: docs/models.yml [hunk 1]
```diff
@@ -258,7 +264,7 @@ committee:
 meeting:
   id: number
   welcome_title: string
-  welcome_text: HTMLVideo
+  welcome_text: HTMLStrictPermissive
```
Comment (normanjaeckel) [inline docs/models.yml:267]: This is wrong.

File: docs/models.yml [hunk 2]
```diff
@@ -840,7 +846,7 @@ topic:
   title:
     type: string
     required: true
-  text: HTMLVideo
+  text: HTMLStrictPermissive
```
Comment (normanjaeckel) [inline docs/models.yml:849]: This is wrong.

**Review by FinnStutzenstein — APPROVED** [review]

File: docs/models.yml (hunk 1 above)
Comment (jsangmeister) [reply docs/models.yml]: ops, dont know how that happened. my bad

**Comment by jsangmeister** [comment]
I think @ostcar (or someone else?) has to fix the validator so this can be merged

**Comment by ostcar** [comment]
The validator failed, because you added a lot of new types.

Can you tell me what types I should add? If I see correctly it is `timestamp`, `HtmlPermissive` and `HTMLStrict`. Is this correct? Anything else?

**Comment by jsangmeister** [comment]
> Can you tell me what types I should add? If I see correctly it is `timestamp`, `HtmlPermissive` and `HTMLStrict`. Is this correct? Anything else?

That's correct. I guess you can remove the old types (`datetime`, `HTML`, `HTMLVideo`) since we don't need them anymore. Also it's all upper case: `HTMLStrict`/`HTMLPermissive`

**Comment by ostcar** [comment]
Done. Please use the version v0.1.1 by changing this line: https://github.com/OpenSlides/OpenSlides/blob/a7248c2399942db9864c538acd3467ccdacdabb2/.github/workflows/models.yml#L14

**Pull request merged**